src/charm.py

```diff
@@ -367,7 +392,11 @@ def _holistic_handler(self, event: EventBase) -> None:
 
     @cached_property
     def _ca_bundle(self) -> str:
-        return TLSCertificates.load(self.certificate_transfer_requirer).ca_bundle
+        v1 = TLSCertificates.load(self.certificate_transfer_requirer_v1).ca_bundle
+        v0 = TLSCertificates.load(self.certificate_transfer_requirer_v0).ca_bundle
+        v1.update(v0)
```

Comment:
`v0` and `v1` are `str`s. Guess it needs something like the following to concatenate them?

```python
return v0 + "\n" + v1
```